pom.xml

```diff
@@ -11,7 +11,7 @@
     </parent>
 
     <artifactId>share.broker.rest</artifactId>
-    <version>8.4.3-SNAPSHOT</version>
+    <version>8.5.0-SNAPSHOT</version>
```

Comment:
if this version doesn't work with the current master branch of the sample.locator without chaning something then the version must be 9.0.0